tests/generics/timplicit_and_explicit.nim

```diff
@@ -34,7 +34,8 @@ block: #15622
   proc test1[T](a: T, b: static[string] = "") = discard
   test1[int64](123)
   proc test2[T](a: T, b: static[string] = "") = discard
-  test2[int64, static[string]](123)
+  doAssert not (compiles do:
+    test2[int64, static[string]](123))
```

Comment:
Huh? Previously this worked and now it doesn't? But it should work as you can pass `123` to `T = int64`. What am I missing?

Reply:
The problem is the extra generic parameter `static[string]`, test2 only accept one generic parameter, and even the implicit generic params which would be the static params don't make sense to be set to literally the type `static[string]`.